content/open-source/octo-sts/faq.md

```diff
@@ -0,0 +1,379 @@
+---
+title: "Octo STS FAQ"
+linktitle: "FAQ"
+type: "article"
+lead: "Frequently asked questions about Octo STS, including troubleshooting, security considerations, and common use cases"
+description: "Learn about Octo STS for GitHub token federation, including setup issues, security best practices, and integration patterns"
+date: 2025-12-22T15:04:05+01:00
+lastmod: 2025-12-22T15:04:05+01:00
+tags: ["octo-sts", "FAQ"]
+draft: false
+images: []
+menu:
+  docs:
+    parent: "octo-sts"
+weight: 60
+toc: true
+---
+
+This page answers frequently asked questions about Octo STS, including setup, security, troubleshooting, and common use cases.
+
+## General Questions
+
+### What is Octo STS?
+
+Octo STS is a GitHub App developer by Chainguard that acts as a Security Token Service for GitHub. It allows workloads with OIDC tokens from various identity providers (GitHub Actions, cloud providers, Kubernetes, etc.) to exchange those tokens for short-lived GitHub access tokens. The primary goal is to eliminate the need for long-lived Personal Access Tokens (PATs).
+
+### Why should I use Octo STS instead of Personal Access Tokens?
+
+Personal Access Tokens are long-lived credentials that pose security risks:
+- They don't expire automatically
+- They're often stored in multiple places
+- If leaked, they provide persistent access
+- They're difficult to rotate across many systems
+
+Octo STS provides:
+- Short-lived tokens that expire automatically
+- No credentials to store in your automation
+- Audit trail of token usage
+- Fine-grained permission control per workload
+
+### How does Octo STS compare to GitHub's built-in GITHUB_TOKEN?
+
+GitHub Actions provides a `GITHUB_TOKEN` automatically, but it has limitations:
```

Comment:
I think this section probably needs some work. I think the main thing is that the GITHUB_TOKEN doesn't have a full set of permissions.